internal/store/pod.go

```diff
@@ -234,7 +234,7 @@ func createPodContainerResourceRequestsFamilyGenerator() generator.FamilyGenerat
 		"kube_pod_container_resource_requests",
 		"The number of requested request resource by a container. It is recommended to use the kube_pod_resource_requests metric exposed by kube-scheduler instead, as it is more precise.",
 		metric.Gauge,
-		basemetrics.ALPHA,
+		basemetrics.BETA,
```

Comment:
Currently, non-STABLE metrics don't have stabilities reflected in the help text. I believe this should, for the most part, be a non-breaking change to implement.

Just a thought, I'll bring it up in the agenda this Thursday.